pkg/cluster/error.go

```diff
@@ -20,7 +20,7 @@ var (
 	errNoBackupExist     = errors.New("no backup exist for disaster recovery")
 	errInvalidMemberName = errors.New("the format of member's name is invalid")
 
-	errUnexpectedUnreadyMember = errors.New("unexpected unready member for selfhosted cluster")
+	errUnexpectedUnreadyMember = errors.New("unready member for selfhosted cluster")
```

Comment:
> why it is not unexpected?

Because we would expect a new etcd pod to take some time to be "ready". Otherwise the logging will say it's "unexpected" and it's confusing to users.